org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -640,38 +640,36 @@ private float IntersectQuad(Vector3 start, Vector3 end, float depth)
         }
 
         /// <summary>
-        /// Registers the duple Collider + XRBaseInteractor as triggering proximity.
+        /// Registers the interactable triggering proximity.
         /// </summary>
-        /// <param name="collider">Collider triggering proximity.</param>
-        /// <param name="xrBaseInteractor">Interactor triggering proximity.</param>
-        public void OnProximityEntered(ProximityEnteredEventArgs proximityEnteredEventArgs)
+        /// <param name="xrBaseInteractable">Interactable triggering proximity.</param>
+        public void OnProximityEntered(ProximityHoverEnteredEventArgs proximityEnteredEventArgs)
         {
-            if (proximityEnteredEventArgs.collider != null &&
-                activeCollidersWithInteractor.Add((proximityEnteredEventArgs.collider, proximityEnteredEventArgs.interactor)) &&
-                activeCollidersWithInteractor.Count >= 1)
+            if (activeDetectors.Add(proximityEnteredEventArgs.NearInteractionModeDetector))
             {
                 UpdateProximityHovered();
             }
         }
 
         /// <summary>
-        /// Unregisters the duple Collider + XRBaseInteractor as triggering proximity.
+        /// Unregisters the interactable triggering proximity.
```

Comment:
"NearInteractionModeDetector" instead of "Interactable"?

Reply:
Went with 'Detector' in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/659/commits/655e552cbcb02e2bdf86e6af4a0a52bebf58f033, resolving.